### PR TITLE
added function isBoolean

### PR DESCRIPTION
**Pull request by JozeFe**
Add new function isBoolean

### CODE REVIEW

File: packages/ramda-extension/src/__tests__/isBoolean-test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { any, equals } from 'ramda';
+import { isBoolean } from '../';
+
+describe('isBoolean', () => {
+	it('should exist', () => {
+		expect(isBoolean).toBeTruthy();
+	});
+
+	it('returns true for Boolean type', () => {
+		expect(any(equals(true), [
```
Comment (jbradle) [inline packages/ramda-extension/src/__tests__/isBoolean-test.js]: Shouldn't all equals true?

File: packages/ramda-extension/src/__tests__/isBoolean-test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+			isBoolean(true),
+			isBoolean(false),
+		])).toBe(true);
+	});
+
+	it('returns false for non-Boolean type', () => {
+		expect(
+			any(equals(true), [
```
Comment (tommmyy) [inline packages/ramda-extension/src/__tests__/isBoolean-test.js]: you can use `R_.equalsToTrue` function :)

File: packages/ramda-extension/src/__tests__/isBoolean-test.js (hunk 1 above)
Comment (JozeFe) [reply packages/ramda-extension/src/__tests__/isBoolean-test.js]: My mistake was in hurry, will fix. I inspired with isNumber-test.js, that will need fix as well.

**Comment by tommmyy** [comment]
@JozeFe any progress?

**Comment by JozeFe** [comment]
@tommmyy Till now I don't have time to fix it, today evening I will push new update. Sorry for to long PR.

**Comment by JozeFe** [comment]
@tommmyy  updated, please check

File: packages/ramda-extension/src/__tests__/isBoolean-test.js (hunk 1 above)
Comment (JozeFe) [reply packages/ramda-extension/src/__tests__/isBoolean-test.js]: @jbradle updated

**Comment by tommmyy** [comment]
@JozeFe thank you for new function!

**Pull request merged**